### DISCUSSION TITLE
How to do basic text formatting in the title, e.g. strikethrough, bold and italics?

### DISCUSSION THREAD

**Discussion by spaceChRiS** [datenstrom/community#938] [Ask a question] [answered]
Hi all,
I have a blog post where I could use a strike-through formatting, but whatever I tried so far does not work: Markdown (I expected it to not work), a html `<s>Text</s>` which does work in the site contents but not in the title, and some different escape sequence combinations (escaping different special characters or even escaping the escape character).
Is there another way to accomplish the strike-through (and other basic formatting such as bold and italics) for a blog post's title?
Thanks!

**Comment by GiovanniSalmeri** [comment]
First of all: could you edit the title of your question in form of a question? (E.g. "How can I etc.?")

Regarding your problem: I had the same problem and finally I wrote the extension [Typography](https://github.com/GiovanniSalmeri/yellow-typography) in order to style the title. I hope it works well with the latest release of Yellow!

**Comment by spaceChRiS** [comment]
This sounds good, I would never have expected an extension called “typography” to add such capabilities to the title. Thanks for the answer, I'll try it out as soon as I can (will need one or two days, as I have an issue with file-system permissions on the server which needs to be sorted before I will be able to install some extension.

Regarding the question title, I made the changes you requested. I basically tried to have the keywords in the title that may lead others with the same issue to this thread, I did not know of a rule that each thread has to be headed by a question.

**Reply by GiovanniSalmeri** [reply, marked as answer]
I am happy if this litle extension will be useful to you! You will find some explications more in https://github.com/datenstrom/yellow/discussions/741

If you give a glance to [https://github.com/GiovanniSalmeri/yellow-typography/blob/main/typography.php lines 271-279](https://github.com/GiovanniSalmeri/yellow-typography/blob/main/typography.php#L271-L279) you will find which markdown characters are supported, among which `~~` for strikethrough <del>(which unfortunately is not supported by the markdown parser used for the content of the pages)</del>.

**Reply by spaceChRiS** [reply]
> (which unfortunately is not supported by the markdown parser used for the content of the pages)

Do you mean the extension is exchanging the markdown parser for the content? Because `~~text~~` does work for me there …

**Reply by GiovanniSalmeri** [reply]
Ooops, you are right. `~~` is not supported by [PHP Markdown Extra](https://michelf.ca/projects/php-markdown/extra/), but has been added in the [Markdown](https://github.com/annaesvensson/yellow-markdown) extension used by Yellow, see https://github.com/annaesvensson/yellow-markdown/blob/main/markdown.php#L3857-L3873
